rust/src/storage/s3/mod.rs

```diff
@@ -856,6 +851,37 @@ impl dyn LockClient {
     }
 }
 
+const S3_GET_INTERNAL_SERVER_ERROR_RETRIES: usize = 10;
```

Comment:
I think this should be an option with default. `10` seems like a fine default value.

Reply:
ok, working on it